pkg/network/node/sdn_controller.go

```diff
@@ -111,6 +111,13 @@ func (plugin *OsdnNode) SetupSDN() (bool, map[string]podNetworkInfo, error) {
 		return false, nil, fmt.Errorf("net/ipv4/ip_forward=0, it must be set to 1")
 	}
 
+	// In OCP 4 apparently we have a lot of rejects.
+	// Default rate mask is 6188, 6160 is default without destination unreachable
```

Comment:
typo. default is 6168. but it would be better to read the current value and unset the relevant bit?

Reply:
Sure, can be done.